### PR TITLE
Componentize PieCharts  in ProgramSunburst and CustomActiveDonut,  About Page in header, body, XoomInOut modules

### PR DESCRIPTION
**Pull request by iksheth**
Componentize PieCharts  in ProgramSunburst and CustomActiveDonut,  About Page in header, body, XoomInOut modules

### CODE REVIEW

File: src/components/PieCharts/CustomActiveDonut/CustomActiveDonutView.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            cy={98}
+            textColor={textColor}
+            innerRadius={45}
+            outerRadius={80}
+            dataKey="value"
+            onMouseEnter={this.onPieEnter}
+            blendStroke
+          >
+            {data.map((entry, index) => (
+              <Cell key={`cell-${index}`} fill={data.length % 2 === 0 ? COLORS_EVEN[index % COLORS_EVEN.length] : COLORS_ODD[index % COLORS_ODD.length]} textColor={textColor} />
```
Comment (AjayDoddapaneni) [inline src/components/PieCharts/CustomActiveDonut/CustomActiveDonutView.js:110]: Can you create COLORS as props so we can change the colors if nothing is specified use these colors as default values?

File: src/components/PieCharts/Donut/DonutView.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent } from 'react';
```
Comment (AjayDoddapaneni) [inline src/components/PieCharts/Donut/DonutView.js:1]: Can you create the stories similar to https://github.com/CBIIT/bento-tools/blob/master/src/stories/index.js#L37 for both donut and sunburst

Comment (iksheth) [reply src/components/PieCharts/Donut/DonutView.js:1]: sure

File: src/components/PieCharts/CustomActiveDonut/CustomActiveDonutController.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```
Comment (AjayDoddapaneni) [inline src/components/PieCharts/CustomActiveDonut/CustomActiveDonutController.js:1]: I don't think we need a controller in here

File: src/components/PieCharts/CustomActiveDonut/CustomActiveDonutView.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
+import React, { PureComponent } from 'react';
+import {
+  PieChart, Pie, Sector, Cell, ResponsiveContainer,
+} from 'recharts';
+
+let COLORS_EVEN = [
```
Comment (AjayDoddapaneni) [inline src/components/PieCharts/CustomActiveDonut/CustomActiveDonutView.js:6]: Can you rename it to DEFAULT_COLORS_EVEN, DEFAULT_COLORS_ODD for more code readability

File: src/components/PieCharts/CustomActiveDonut/CustomActiveDonutView.js [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    const {
+      data: DataObj, textColor, colors,
+    } = this.props;
+    const data = DataObj.map((obj) => ({
+      name: obj.group,
+      value: obj.subjects,
+    }));
+
+    if (colors) {
+      COLORS_EVEN = colors.even;
```
Comment (AjayDoddapaneni) [inline src/components/PieCharts/CustomActiveDonut/CustomActiveDonutView.js:92]: you can use a ternary operator to check these conditions

File: src/components/PieCharts/Donut/DonutController.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```
Comment (AjayDoddapaneni) [inline src/components/PieCharts/Donut/DonutController.js:1]: Same in here we don't need a controller in here we can directly export view and use the controller in the data commons

File: src/components/PieCharts/ProgramSunburst/ProgramSunburstView.js [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+          <div className={classes.title}>
+            {title}
+          </div>
+        </div>
+      </>
+    );
+  }
+}
+
+export default withStyles(styles)(ProgramSunburst);
```
Comment (AjayDoddapaneni) [inline src/components/PieCharts/ProgramSunburst/ProgramSunburstView.js:162]: Can we use injectSheet from react-jss so we don't need a dependency on MUI for this component

File: src/components/about/aboutHeaderView.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+    height: '65px',
+    width: '252px',
+    color: '#0077E3',
+    fontFamily: 'Lato',
+    fontSize: '35px',
+    fontWeight: 'bold',
+  },
+});
+
+export default withStyles(styles)(AboutHeader);
```
Comment (AjayDoddapaneni) [inline src/components/about/aboutHeaderView.js:37]: Can use injectSheet in here

File: src/components/about/xoomInOutView.js [hunk 9]
```diff
@@ -0,0 +1,154 @@
+import React from 'react';
```
Comment (AjayDoddapaneni) [inline src/components/about/xoomInOutView.js:1]: Can you export this as a separate component, so we can use this component independently in future

File: src/components/PieCharts/ProgramSunburst/ProgramSunburstController.js [hunk 10]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```
Comment (AjayDoddapaneni) [inline src/components/PieCharts/ProgramSunburst/ProgramSunburstController.js]: I don't think we need a controller

File: src/components/about/xoomInOutView.js (hunk 9 above)
Comment (iksheth) [reply src/components/about/xoomInOutView.js:1]: @AjayDoddapaneni Yes it's already exported as separate component.

**Comment by AjayDoddapaneni** [comment]
Looks Good!! Can you fix the conflixts

**Review by AjayDoddapaneni — APPROVED** [review]

**Pull request merged**